adagrams/game.py

```diff
@@ -1,11 +1,104 @@
+from string import ascii_uppercase
+from random import randint , choice
```

Comment:
Extra whitespace after `randint` which should be removed. Since `randint` is not being accessed we should remove unused imports to keep our project neat. 

Also, usually we include a blank line after imports so we can remove the two extra blank lines on lines 4-5.

It's a small thing, but as you write more code (and eventually contribute to a large codebase) it's important to be consistent with whitespaces and blank lines so that the whole project stays neat. Here's the official Python style guide that covers [whitespaces](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements) and here's the section on [blank lines](https://peps.python.org/pep-0008/#blank-lines).

Reply:
Thank you! Also, I left the improt of choice function to let the commented out function be functional

Reply:
In the future, you can leave me a comment in the PR to show me an alternative approach. That way you don't need to have long sections of commented out code in your project.

Reply:
Okay, I’ll keep that in mind.